src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/InvokeRestMethodCommand.Common.cs

```diff
@@ -268,7 +263,7 @@ private static bool TryConvertToXml(string xml, out object doc, ref Exception ex
                 XmlReaderSettings settings = GetSecureXmlReaderSettings();
                 XmlReader xmlReader = XmlReader.Create(new StringReader(xml), settings);
 
-                var xmlDoc = new XmlDocument();
+                XmlDocument xmlDoc = new();
```

Comment:
Just to call out that we don't recommend this style change in the existing code base (_for_ `var name = new Ctor`). The original `var xmlDoc = new XmlDocument()` doesn't have any readability issue, so let's not change it to avoid code churn.

However, for new code, you can use either style.